### PR TITLE
First commit - solve practice

### PR DESCRIPTION
**Pull request by lourdesVillca**

### CODE REVIEW

File: src/main/java/org/fundacionjala/StringTask/SearchEngine.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Created by LourdesVillca on 9/2/2016.
+ */
+public class SearchEngine {
+    
+    public static int find(String needle, String haystack) {
+            String regex = "\\Q" + needle.replaceAll("_", "\\\\E.\\\\Q") + "\\E";
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/StringTask/SearchEngine.java]: This should be final

File: src/test/java/org/fundacionjala/StringTask/WildsTest.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package org.fundacionjala.StringTask;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by LourdesVillca on 9/2/2016.
+ */
+public class WildsTest {
+    String haystack = "Once upon a midnight dreary, while I pondered, weak and weary";
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/StringTask/WildsTest.java]: This should be a constant.
private static final

File: src/main/java/org/fundacionjala/StringTask/SearchEngine.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package org.fundacionjala.StringTask;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Created by LourdesVillca on 9/2/2016.
+ */
+public class SearchEngine {
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/StringTask/SearchEngine.java:9]: Add a private constructor.

File: src/test/java/org/fundacionjala/StringTask/WildsTest.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    @Test
+    public void normalSearchTest(){
+        assertEquals(0,SearchEngine.find("Once", haystack));
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/StringTask/WildsTest.java]: Give format to the class

**Comment by carledriss** [comment]
@lourdesVillca 
Let me know when you have fixed the observations.

**Comment by lourdesVillca** [comment]
@carledriss ready to code review

**Comment by carledriss** [comment]
Looks good to me!

**Pull request merged**